<launch>
    <arg name="veh"  doc="Name of vehicle. ex: megaman"/>
    <arg name="pkg_name" value="object_detection"/>
    <arg name="node_name" default="object_detection_node"/>
    <arg name="local" default="1"/>
    <arg name="debug" default="false"/>

    <group ns="$(arg veh)">
        <remap from="object_detection_node/image_compressed" to="camera_node/image/compressed/"/>
        <node if="$(arg local)" name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen">
            <rosparam command="load" file="$(eval find(pkg_name) + '/default.yaml')"/>
        </node>
    </group>
</launch>

<!--launch-prefix="xterm -e python -m pdb"-->